torchvision/models/detection/transform.py

```diff
@@ -30,17 +30,19 @@ def __init__(self, min_size, max_size, image_mean, image_std):
         self.image_std = image_std
 
     def forward(self, images, targets=None):
+        transformed_images = []
         for i in range(len(images)):
-            image = images[i]
+            image = images[i].clone()
```

Comment:
This clone is not necessary because `normalize` is done out-of-place

Reply:
removed